include/pressio/ops.hpp

```diff
@@ -158,6 +159,7 @@ template<class ...> struct matching_extents;
 #include "ops/epetra/ops_deep_copy.hpp"
 #include "ops/epetra/ops_scale.hpp"
 #include "ops/epetra/ops_set_zero.hpp"
+#include "ops/epetra/ops_scale.hpp"
```

Comment:
scale is included twice

Reply:
thanks for catching this!
fixed on bfa5ccee99577af5e5998e33253a6a826a0186a3